api/v1alpha3/machinedeployment_webhook_test.go

```diff
@@ -45,6 +44,8 @@ func TestMachineDeploymentDefault(t *testing.T) {
 	g.Expect(md.Spec.Strategy.RollingUpdate).ToNot(BeNil())
 	g.Expect(md.Spec.Strategy.RollingUpdate.MaxSurge.IntValue()).To(Equal(1))
 	g.Expect(md.Spec.Strategy.RollingUpdate.MaxUnavailable.IntValue()).To(Equal(0))
+	g.Expect(md.Spec.Selector.MatchLabels).To(HaveKeyWithValue(clusterv1.ClusterLabelName, "test-md"))
+	g.Expect(md.Spec.Template.Labels).To(HaveKeyWithValue(clusterv1.ClusterLabelName, "test-md"))
```

Comment:
@hpandeycodeit You probably need to remove `clusterv1.ClusterLabelName` and add `ClusterLabelName` instead.

Reply:
thanks! changed it :)